docs/src/apimanual.md

```diff
@@ -893,27 +893,55 @@ Avoid storing extra copies of the problem when possible. This means that solver
  - `@variable(m, x >= 1)` becomes `SingleVariable`-in-`GreaterThan`.
 
-One notable difference is that a variable with an upper and lower bound is translated into two constraints, rather than an interval. i.e.:
- - `@variable(m, 0 <= x <= 1)` becomes `SingleVariable`-in-`LessThan` *and* `SingleVariable`-in-`GreaterThan`.
-
-Therefore, if a solver wrapper does not support `ScalarAffineFunction`-in-`LessThan` constraints, users will not be able to write: `@constraint(m, 2x + y <= 10)` in JuMP. With this in mind, developers should support all the constraint types that they want to be usable from JuMP. That said, from the perspective of JuMP, solvers can safely choose to not support the following constraints:
-
-- `ScalarAffineFunction` in `GreaterThan`, `LessThan`, or `EqualTo` with a nonzero constant in the function. Constants in the affine function should instead be moved into the parameters of the corresponding sets.
-
-- `ScalarAffineFunction` in `Nonnegative`, `Nonpositive` or `Zeros`. Alternative constraints are available by using a `VectorAffineFunction` with one output row or `ScalarAffineFunction` with `GreaterThan`, `LessThan`, or `EqualTo`.
-
-- Two `SingleVariable`-in-`LessThan` constraints applied to the same variable (similarly with `GreaterThan`). These should be interpreted as variable bounds, and each variable naturally has at most one upper or lower bound.
+One notable difference is that a variable with an upper and lower bound is
+translated into two constraints, rather than an interval. i.e.:
+ - `@variable(m, 0 <= x <= 1)` becomes `SingleVariable`-in-`LessThan` *and*
+   `SingleVariable`-in-`GreaterThan`.
+
+Therefore, if a solver wrapper does not support
+`ScalarAffineFunction`-in-`LessThan` constraints, users will not be able to
+write: `@constraint(m, 2x + y <= 10)` in JuMP. With this in mind, developers
+should support all the constraint types that they want to be usable from JuMP.
+That said, from the perspective of JuMP, solvers can safely choose to not
+support the following constraints:
+
+- `ScalarAffineFunction` in `GreaterThan`, `LessThan`, or `EqualTo` with a
+  nonzero constant in the function. Constants in the affine function should
+  instead be moved into the parameters of the corresponding sets.
+
+- `ScalarAffineFunction` in `Nonnegative`, `Nonpositive` or `Zeros`. Alternative
+  constraints are available by using a `VectorAffineFunction` with one output
+  row or `ScalarAffineFunction` with `GreaterThan`, `LessThan`, or `EqualTo`.
+
+- Two `SingleVariable`-in-`LessThan` constraints applied to the same variable
+  (similarly with `GreaterThan`). These should be interpreted as variable
+  bounds, and each variable naturally has at most one upper or lower bound.
+
+In addition, when creating functions such as `ScalarAffineFunction` and
+`ScalarQuadraticFunction`, JuMP will aggregate duplicate coefficients. For
+example, `2x + x` will be passed as `3x`, a `ScalarAffineFunction` with one
```

Comment:
Bridges can create duplicate coefficients so solvers should still support them. They can call `MOIU.canonical(f)` to remove them.